app/components-react/windows/go-live/useGoLiveSettings.ts

```diff
@@ -46,6 +47,11 @@ export class GoLiveSettingsModule extends StreamInfoView<IGoLiveSettingsState> {
    */
   async prepopulate() {
     await Services.StreamingService.actions.return.prepopulateInfo();
+
+    // TODO: investigate why mutations from `StreamingService.prepopulateInfo` are not applied at this moment
+    // use `sleep` to wait for mutations as a temporary fix
+    await Utils.sleep(100);
```

Comment:
This makes me a bit nervous.
A) it's bad for perceived perf, and
B) 100ms may not be enough for everyone

Reply:
A) It takes about 1s to prepopulate platforms first time for me. Additional 100ms doesn't make a big difference
B) I took 100ms as a huge reserve. Actually, we need only time for executing one IPC request here which is usually 2 - 10ms
I agree that looks like a hack. We should fix an original issue with incorrect mutations order here